skimage/filters/_frangi.py

```diff
@@ -114,7 +114,7 @@ def frangi(image, scale_range=(1, 10), scale_step=2, beta1=0.5, beta2=15,
            vessel enhancement filtering," In LNCS, vol. 1496, pages 130-137,
            Germany, 1998. Springer-Verlag.
     .. [2] Kroon, D.J.: Hessian based Frangi vesselness filter.
-    .. [3] http://mplab.ucsd.edu/tutorials/gabor.pdf.
+    .. [3] https://web.archive.org/web/20180127125930/http://mplab.ucsd.edu/tutorials/gabor.pdf
```

Comment:
I'm wondering how reliable web.archive.org links are :)...

An alternative could be a ResearchGate version, uploaded by the author - https://www.researchgate.net/publication/265064863_Tutorial_on_Gabor_Filters .

Reply:
I think it is less reliable... webarchive exists since a while

Reply:
Oh yeah, no way should we rely on ResearchGate, which is being sued by Elsevier. (Wrongly, but it's not like wrong things don't happen in law and life. =P) Web Archive is about as reliable as Wikipedia I think: a very valuable public good that is unlikely to go away without a fight.

Reply:
@jni So, ready to be merged? :)